src/playground/without-layout/search/search-with-input-event.component.ts

```diff
@@ -0,0 +1,41 @@
+import { Component, OnInit } from '@angular/core';
+import {NbSearchService} from '@nebular/theme';
+import {debounceTime, distinctUntilChanged} from 'rxjs/operators';
+
+@Component({
+  selector: 'nb-search-with-input-event',
+  template: `
+    <nb-layout>
+      <nb-layout-header fixed>
+        <nb-search type="rotate-layout" tag="search-input-event"></nb-search>
+      </nb-layout-header>
+      <nb-sidebar>
+      </nb-sidebar>
+
+      <nb-layout-column>
+        <nb-card accent="info">
+          <nb-card-header>You searched for:</nb-card-header>
+          <nb-card-body>
+            <h3>{{ value || '-' }}</h3>
+          </nb-card-body>
+        </nb-card>
+      </nb-layout-column>
+    </nb-layout>
+
```

Comment:
unnecessary empty line

Reply:
Removed in f5d96dc8807911f3a5d617fdf4c768042bbe5aac